cmd/katib-controller/v1beta1/main.go

```diff
@@ -131,27 +133,40 @@ func main() {
 		CertDir: consts.CertDir,
 	})
 
+	if err := mgr.AddReadyzCheck("webhook", hookServer.StartedChecker()); err != nil {
+		log.Error(err, "Unable to add readyz check cert")
+		os.Exit(1)
+	}
+
 	ctx := signals.SetupSignalHandler()
 	certsReady := make(chan struct{})
-	defer close(certsReady)
 
 	// The setupControllers will register controllers to the manager
 	// after generated certs for the admission webhooks.
 	go setupControllers(mgr, certsReady, hookServer)
 
 	if initConfig.CertGeneratorConfig.Enable {
+		err := mgr.AddReadyzCheck("cert", func(_ *http.Request) error {
+			select {
+			case <-certsReady:
+				return nil
+			default:
+				return fmt.Errorf("cert not ready")
```

Comment:
Please can we make error messages consistent with Training Operator: https://github.com/kubeflow/training-operator/blob/master/cmd/training-operator.v1/main.go#L242